README.md

```diff
@@ -43,6 +43,42 @@ This will output the following snippet in the generated XML
 
 The plugin will replace `@JIRA_` with `AltID_` in the generated XML.
 
+## Using `<tags>` element
+
+`@` tags in the feature file which are not recognised as requirements (see above), will be inserted into the XML results file as tags to be attached to the Testcase in Zephyr.
+
+For example:
+
+```gherkin
+@JIRA_XYZ-1 @JIRA_XYZ-1 @smoke-test
+Feature: something
+```
+
+This will collect the first two tags as requirements and the last `@` tag will be a tag for the zephyr tescase. 
+
+Here is an example of a single tag in the XML file:
+```xml
+<tags>                                    ---//  tags: parent element
+<tag>Feature1</tag>             ---// tag : child element 
+</tags>  
+```
+
+Here is an example of a multiple tags in the XML file:
+```xml
+<tags>
+<tag>BVT1</tag>
+<tag>BVT2</tag>
+</tags>
+```
+
+In conjection with having this data in the XML file, we need to ensure the custom parser is able to read the tag. 
+
+Sample line which we should use in the template to add tags 
```

Comment:
Is this paragraph relevant to users of this library? This sounds more like technical details about the system that will process the generated XML.

I think this information is confusing for the users of this library and should go elsewhere.